packages/wallet/backend/src/outgoingPayment/service.ts

```diff
@@ -17,11 +17,16 @@ export class OutgoingPaymentService implements IOutgoingPaymentService {
     const quote = await this.deps.rafikiClient.getQuote(quoteId)
     const paymentPointerId = quote.paymentPointerId
 
-    const incomingPayment =
-      await this.deps.incomingPaymentService.getPaymentDetailsByUrl(
-        quote.receiver
-      )
-    const { description } = incomingPayment
+    let description
+    try {
+      const incomingPayment =
+        await this.deps.incomingPaymentService.getPaymentDetailsByUrl(
+          quote.receiver
+        )
+      description = incomingPayment.description
+    } catch (_e) {
+      // @todo: find another way to get payment description
```

Comment:
@raducristianpopa `getPaymentDetailsByUrl` searches for a local incoming payment and if it does not exist it throws 404
in case of external payments there is no local incoming payment. The external outgoing payments won't have a description for now.